vendor/github.com/stretchr/testify/vendor/github.com/stretchr/objx/codegen/types_list.txt

```diff
@@ -0,0 +1,20 @@
+Interface,interface{},"something",nil,Inter
```

Comment:
I'm not sure, I just did govendor sync, so I think they were outdated or manually changed?